tests/queries/0_stateless/01275_parallel_mv.reference

```diff
@@ -10,14 +10,14 @@ insert into testX select number from numbers(10) settings
     optimize_trivial_insert_select=0,
     max_insert_threads=0; -- { serverError FUNCTION_THROW_IF_VALUE_IS_NON_ZERO }
 system flush logs;
-select arrayUniq(thread_ids) from system.query_log where
+select peak_threads_usage from system.query_log where
     current_database = currentDatabase() and
     type != 'QueryStart' and
     query like '%insert into testX %' and
     Settings['parallel_view_processing'] = '0' and
     Settings['optimize_trivial_insert_select'] = '0' and
     Settings['max_insert_threads'] = '0';
-2
+5
```

Comment:
I do not understand why. Why count all participated threads bigger that maximum simultaneous running threads?
Do we sure that it is counted correctly?

Reply:
https://github.com/ClickHouse/ClickHouse/issues/66116#issuecomment-2211753262

Reply:
I would rather wait with merging this fix until we understand it.
May be we need fix the counter instead using wrong one as a fix for something.

Reply:
Fix https://github.com/ClickHouse/ClickHouse/pull/66230

Reply:
Rebased on top of #66230